src/order-history/OrderHistoryPage.jsx

```diff
@@ -91,32 +91,34 @@ class OrderHistoryPage extends React.Component {
 
   renderOrdersTable() {
     return (
-      <Table
+      <DataTable
         className="order-history table-bordered"
+        itemCount={this.props.orders.length}
         data={this.getTableData()}
```

Comment:
I've done that in `ReceiptPage.jsx` when I learned about `Cell` from you 😬, I will update this here, good catch because I was just updating the deprecated table component to use `DataTable`, thanks!